loops-and-arrays.js

```diff
@@ -6,6 +6,18 @@
 
 const names = ["Henk", "Piet", "Fred", "Joop"];
 
+console.log("originele waardes:");
+for (let i = 0; i < names.length; i++) {
+    console.log(names[i]);
+}
+
+console.log("aangepaste waardes:");
+for (let i = 0; i < names.length; i++) {
+    const nickName = names[i];
+    names[i] = nickName + "je";
+    console.log(names[i]);
```

Comment:
Ook hier de opmerking dat het eenvoudiger had gekund. Persoonlijk denk ik ook dat het goed is om de bewerkingen te scheiden van het 'afdrukken', zeker als we straks meer met functies gaan werken. Dat maakt de code leesbaarder.